### PR TITLE
[website]Add new files on tutorials and community part

### PR DESCRIPTION
**Pull request by StrayBird-ATSH**
## Description ##
According to issue (#1081), the main structure of the website examples part needs to be refactored. Currently this pull request deals with tutorials and community part of this issue. _This pull request deals with almost all the remaining work that needs to do in the tutorials and community part except major enhancements._ All existing structure of the examples part of the website are modified accordingly. For major enhancements, like the gallery, will be added later.

### Essentials ###
- [x] PR's title starts with a category (e.g. [BUGFIX], [MODEL], [TUTORIAL], [FEATURE], [DOC], etc)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage
- [x] Code is well-documented

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

cc @dmlc/gluon-nlp-team

### CODE REVIEW

**Comment by mli** [comment]
Job PR-1162/2 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-1162/2/index.html

**Comment by mli** [comment]
Job PR-1162/4 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-1162/4/index.html

File: docs/examples/language_model/language_model2.md [hunk 1]
```diff
@@ -0,0 +1,292 @@
+# Train your own LSTM based Language Model
```
Comment (eric-haibin-lin) [inline docs/examples/language_model/language_model2.md]: Could you rename them to:
- `language_model/language_model.md ` -> `language_model/use_pretrained_lm.md `
- `language_model/language_model2.md ` -> `language_model/train_language_model.md `

File: docs/examples/index.rst [hunk 2]
```diff
@@ -3,6 +3,30 @@ Tutorials
...
+      :link: notes/vocab_emb.html
+
+      Learn how to write simple code to create index for tokens.
+
+
+.. toctree::
+   :hidden:
+   :maxdepth: 2
+
+   notes/index
```
Comment (eric-haibin-lin) [inline docs/examples/index.rst]: I see "Notes" from the sidebar. Could you update https://github.com/dmlc/gluon-nlp/blob/600691b2e06b8f317d04ee5bf27014f37ad15a0c/docs/examples/notes/index.rst#notes to replace "Notes" with "Data Loading and Vocabularies"

File: docs/examples/index.rst [hunk 3]
```diff
@@ -3,6 +3,30 @@ Tutorials
 
 Interested in getting started in a new NLP area? Here are some tutorials to help get started.
 
+Dataset and Vocabularies
```
Comment (eric-haibin-lin) [inline docs/examples/index.rst]: Could you change it to `Data Loading and Vocabularies`

File: docs/api/index.rst [hunk 4]
```diff
@@ -4,5 +4,4 @@ API Documentation
 .. toctree::
    :maxdepth: 2
 
-   notes/index
```
Comment (eric-haibin-lin) [inline docs/api/index.rst]: Can we remove "Package Reference" button and directly show `gluonnlp.xx` under API Documentation? We can make it flat. 
Screen Shot 2020-02-18 at 10 22 22 AM

**Comment by mli** [comment]
Job PR-1162/7 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-1162/7/index.html

**Comment by mli** [comment]
Job PR-1162/8 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-1162/8/index.html

**Comment by mli** [comment]
Job PR-1162/10 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-1162/10/index.html

**Pull request merged**